db/python/layers/project_insights.py

```diff
@@ -528,24 +537,24 @@ async def _crams_by_project_id_and_seq_fields(
         sequencing_types: list[SequencingType],
     ) -> dict[ProjectSeqTypeTechnologyKey, list[SequencingGroupInternalId]]:
         _query = """
-        SELECT
-            a.project,
-            sg.type as sequencing_type,
-            sg.technology as sequencing_technology,
-            GROUP_CONCAT(DISTINCT asg.sequencing_group_id) as sequencing_group_ids
-        FROM
-            analysis a
-            LEFT JOIN analysis_sequencing_group asg ON a.id = asg.analysis_id
-            LEFT JOIN sequencing_group sg ON sg.id = asg.sequencing_group_id
-        WHERE
-            a.project IN :projects
-            AND sg.type IN :sequencing_types
-            AND a.type = 'CRAM'
-            AND a.status = 'COMPLETED'
-        GROUP BY
-            a.project,
-            sg.type,
-            sg.technology;
+SELECT
+    a.project,
+    sg.type as sequencing_type,
+    sg.technology as sequencing_technology,
+    GROUP_CONCAT(DISTINCT asg.sequencing_group_id) as sequencing_group_ids
+FROM
+    analysis a
+    LEFT JOIN analysis_sequencing_group asg ON a.id = asg.analysis_id
+    LEFT JOIN sequencing_group sg ON sg.id = asg.sequencing_group_id
+WHERE
+    a.project IN :projects
+    AND sg.type IN :sequencing_types
+    AND a.type = 'CRAM'
+    AND a.status = 'COMPLETED'
+GROUP BY
+    a.project,
+    sg.type,
+    sg.technology;
```

Comment:
Might be better to move this to the `tables` submodule since most SQL logic for other entities resides there and the layer usually calls the table methods? The SQL was already here so I'm neutral about moving it, just my OCD ticking :)